core/src/classes/plugin.ts

```diff
@@ -460,7 +461,7 @@ export interface SourceFilterMethod<AppConnection = any> {
 
 export interface SourceFilterMethodResponseRow {
   key: string;
-  ops: Array<string>;
+  ops: Array<keyof typeof FilterOpsDescriptions>;
```

Comment:
There were quite a few place to change "string" into true lists of the available options from our dictionaries and descriptions.

Reply:
```suggestion
  ops: (keyof typeof FilterOpsDescriptions)[];
```
Based on our future design doc!